modules/sdk-core/src/bitgo/keychain/keychains.ts

```diff
@@ -296,7 +296,12 @@ export class Keychains implements IKeychains {
     let MpcUtils;
     switch (params.multisigType) {
       case 'tss':
-        MpcUtils = this.baseCoin.getMPCAlgorithm() === 'ecdsa' ? ECDSAUtils.EcdsaUtils : EDDSAUtils.default;
+        MpcUtils =
+          this.baseCoin.getMPCAlgorithm() === 'eddsa'
+            ? EDDSAUtils.default
+            : params.walletVersion === 5
```

Comment:
walletVersion is specific to eth, it will not work for cosmos chains. What about using mpcTypeVersion instead?